doc/nrf/ug_thread_supported_features.rst

```diff
@@ -35,6 +35,12 @@ In |NCS|, you can choose which version of the Thread protocol to use in your app
 By default, |NCS| supports Thread 1.1, but you can enable and configure Thread 1.2 by using :ref:`dedicated options <thread_ug_thread_1_2>`.
 
 .. note::
-    Not all Thread 1.2 functionalities are currently supported.
+    All Thread 1.2 mandatory functionalities are currently implemented, execept for full Border Router support.
     See :ref:`thread_ug_thread_1_2` for the list of 1.2 features that are currently available in |NCS|, with information about how to enable them.
     Currently, the :ref:`ot_cli_sample` sample is the only sample that provides an :ref:`ot_cli_sample_thread_v12`.
+
+    There are known limitations on the Thread 1.2 support:
```

Comment:
I would suggest a subsection:
```
Limitations for Thread 1.2 support
=====================

<content of lines 38-40>

The Thread 1.2 Specification support has the following limitations:

* The current implementation does not guarantee that all retransmitted frames will be secured when using the radio driver transmission security capabilities.
  For this reason , OpenThread retransmissions are disabled by default when the :option:`CONFIG_NRF_802154_ENCRYPTION` Kconfig option is enabled.
You can enable the retransmissions at your own risk.
* Due to code size limitation, the combination of complete set of Thread 1.2 features with the Bluetooth LE multiprotocol support is not possible for the nRF52833 DKs.
```
Are these limitations or known issues that should be tracked on the known issues page?

Reply:
> Are these limitations or known issues that should be tracked on the known issues page?
I would say that's the case for the first one at least.